docs/contributing.md

```diff
@@ -1,12 +1,34 @@
----
-editLink: true
----
-
-# Contributing Guideline
+# How to Participate
+Development of the Open DC Grid Standard and the associated reference implementations is a collaborative effort driven by volunteers.
+We welcome anyone to offer ideas and technical contributions.
 
 We welcome every sort of contribution, from error reports to improvements of the document or entirely new ideas or chapters.
 
-## GitHub workflow
+## Contact us
+<a href="mailto:opendcgrid@gmail.com" target="_top"><img style="float: left;" src="./images/Letter.svg" hspace="0" width="80" height="40" alt="Open DC Grid Email"></img></a>You can contact us by email: <opendcgrid@gmail.com>
+
+## Attend a meeting
+<a href="https://join.freeconferencecall.com/jlgula" target="_blank"><img style="float: left;" src="./images/FreeConferenceCall.png" hspace="10" width="64" height="64" alt="Link to Freeconferencecall meeting page"></img></a>
+The project holds monthly web-based teleconferences using the conferencing service [freeconferencecall.com](https://www.freeconferencecall.com) to discuss the status and present new ideas. 
+A login to the service is not required but downloading their free app can give you a better experience.
+We record the meetings so people can review them later and provide a link to any materials presented.
+Meeting details are available on the [Meetings](./meetings) page of this web site.
+
+##  Visit us on GitHub
+<a href="https://github.com/open-dc-grid" target="_blank"><img style="float: left;" src="./images/GitHub-Mark-64px.png" hspace="10" alt="Link to Open DC Grid GitHub page"></img></a>
+All project materials are publically accessible on the software development platform [GitHub](https://github.com/) as a collection of repositories under the [Git](https://git-scm.com/) source control system for the organization [open-dc-grid](https://github.com/open-dc-grid).
+A GitHub login is not required to view or download the materials but is required if you wish to propose new material or changes to existing materials. GitHub logins are free.
+
+Note that Git is supported by many of the popular integrated development environments such as Windows [Visual Studio](https://visualstudio.microsoft.com/) (including the free [Community Edition](https://visualstudio.microsoft.com/vs/community/), 
+[Apple XCode](https://developer.apple.com/xcode/), [Eclipse](https://www.eclipse.org/ide/) and others. Using an IDE can be easier for people unused to command line tools but many developers prefer to use Git directly from the command line with their favorite editor.
+
+Available Open DC Grid materials include:
+
+* the text of the [Open DC Grid Standard](./standard) in the repository [standard](https://github.com/open-dc-grid/standard)
+* reference designs for conforming products and implementation suggestions in the repository [implementation](https://github.com/open-dc-grid/implementation)
```

Comment:
Well, there are no reference designs yet, so people could be disappointed. I would also understand the implementation section at the moment as a place where we can put suggestions for certain parts of a device, e.g. the grid interface short circuit protection. It will still take a while until we have proper reference designs. Or do you think a reference design can also refer to a small example schematic?